sass/layout/_headers.scss

```diff
@@ -290,6 +289,11 @@
         color: $dark-gray;
     }
 
+    .BadgeContainer {
```

Comment:
This is the example of problem that I want to avoid, here I would search for a `BadgeContainer` component that doesn't exist. I would expect to see here `.Badge` and that's it.